rb/lib/selenium/webdriver/firefox/binary.rb

```diff
@@ -170,8 +170,8 @@ def windows_path
 
           def macosx_path
             path = "/Applications/Firefox.app/Contents/MacOS/firefox-bin"
-            path = "~/Applications/Firefox.app/Contents/MacOS/firefox-bin" unless File.exist?(path)
-            path = Platform.find_binary("firefox-bin") unless File.exist?(path)
+            path = "~/Applications/Firefox.app/Contents/MacOS/firefox-bin" unless File.exist?(File.expand_path(path))
```

Comment:
I don't think this one needs to be changed since the only previous path is already an absolute one.